app/src/main/java/com/SecUpwN/AIMSICD/smsdetection/SmsDetector.java

```diff
@@ -504,4 +313,69 @@ public void onServiceDisconnected(ComponentName arg0) {
             mBound = false;
         }
     };
+
+    private String findSmsData(String[] prebuffer,String[] postbuffer){
```

Comment:
I give up cant deal with someone this ocd about code. Also delete my sms detection please since the code is not to scratch. This project is dead with people like tou giving people a hard time and not trying to help by hitting the edited button. All you seem to do is add more whitespace and delete valuable comments and code that is needed for future work.